lib/src/constants.dart

```diff
@@ -2,6 +2,6 @@ class Timeouts {
   static const connection = Duration(seconds: 10);
   static const debounce = Duration(milliseconds: 100);
   static const publish = Duration(seconds: 10);
-  static const iceConnection = Duration(seconds: 10);
+  static const peerConnection = Duration(seconds: 10);
```

Comment:
If this is the timeout for initial media connection, it was bumped to 15 seconds in JS SDK to deal with Safari taking a long time to connect in long delay networks. Would be good to have a consistent time out across all clients.